<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowCntr</class>
 <widget class="QMainWindow" name="WindowCntr">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>290</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>341</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>341</width>
    <height>290</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Roboto</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>EMBO - Counter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/main/img/icon.png</normaloff>:/main/img/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_enable">
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>185</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {    
    padding: 0.2em 0.2em 0.3em 0.2em;
    border: 1px solid rgb(150, 150, 150);
	border-radius: 5px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:2,
                                  stop:0 #FFFFFF, stop:0.1 #FFFFFF, stop:1  #7B7B7B);
    color: black;
	text-align:left;
	padding-left:13px;
}

QPushButton:hover:!pressed
{
	border: 1px solid rgb(150, 150, 150);
	border-radius: 5px;
 	background: #FFFFFF;
	color: #29715c;
}

QPushButton:pressed
{
	border: 2px solid rgb(150, 150, 150);
	border-radius: 5px;
	color:#2f836a;
 	background: #FFFFFF;
}</string>
    </property>
    <property name="text">
     <string> Counter OFF</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources/resources.qrc">
      <normaloff>:/main/img/power-off-gray2.png</normaloff>
      <disabledoff>:/main/img/power-off-gray2.png</disabledoff>
      <disabledon>:/main/img/power-off-gray2.png</disabledon>:/main/img/power-off-gray2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>95</x>
      <y>185</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {    
    padding: 0.2em 0.2em 0.3em 0.2em;
    border: 1px solid rgb(150, 150, 150);
	border-radius: 5px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:2, stop:0 #FFFFFF, stop:0.1 #FFFFFF, stop:1  #69D769);
    color: black;
}

QPushButton:hover:!pressed
{
	border: 1px solid rgb(150, 150, 150);
	border-radius: 5px;
 	background: #FFFFFF;
	color:#2f836a;
}

QPushButton:pressed
{
	border: 2px solid rgb(150, 150, 150);
	border-radius: 5px;
	color:#2f836a;
 	background: #FFFFFF;
}

QPushButton { 
	text-align:left;
	padding-left:13px;
}</string>
    </property>
    <property name="text">
     <string> Counter ON</string>
    </property>
    <property name="icon">
     <iconset resource="../../resources/resources.qrc">
      <normaloff>:/main/img/power-off-green3.png</normaloff>
      <disabledoff>:/main/img/power-off-green3.png</disabledoff>:/main/img/power-off-green3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>234</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>1</y>
      <width>321</width>
      <height>165</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {background-color: rgb(220,220,220);border: 1px solid rgb(150,150,150); border-radius: 5px;  margin-top: 0.5em;}; 
QGroupBox::title {subcontrol-origin: margin;left: 10px;padding: 0 3px 0 3px;}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Period:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_period">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Mono Medium</family>
       <pointsize>16</pointsize>
       <weight>62</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
	background-color: rgba(240,240,240, 1.0);border: 1px solid gray; border-radius:10px;
	text-align:right;font-family:'Roboto Mono Medium'; font-size:16pt; font-weight:500;
}

QTextBrowser:disabled {
	background-color: transparent;
	border-color: transparent; border-width: 1px solid gray; border-style:inset; border-radius:10px;
	text-align:right;font-family:'Roboto Mono Medium'; font-size:16pt; font-weight:500;
}</string>
     </property>
     <property name="markdown">
      <string/>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Mono Medium'; font-size:16pt; font-weight:496; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Light</family>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Frequency:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_freq">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Roboto Mono Medium</family>
       <pointsize>16</pointsize>
       <weight>62</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor" stdset="0">
      <cursorShape>IBeamCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
	background-color: rgba(240,240,240, 1.0);border: 1px solid gray; border-radius:10px;
	text-align:right;font-family:'Roboto Mono Medium'; font-size:16pt; font-weight:500;
}

QTextBrowser:disabled {
	background-color: transparent;
	border-color: transparent; border-width: 1px solid gray; border-style:inset; border-radius:10px;
	text-align:right;font-family:'Roboto Mono Medium'; font-size:16pt; font-weight:500;
}</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Roboto Mono Medium'; font-size:16pt; font-weight:496; font-style:normal;&quot;&gt;
&lt;p align=&quot;right&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>115</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(150,150,150,0.6);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <zorder>textBrowser_period</zorder>
    <zorder>textBrowser_freq</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>line_2</zorder>
   </widget>
   <widget class="QRadioButton" name="radioButton_fast">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>135</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:checked {
	background-color: rgba(245,245,245, 1.0);
	border: 0px; 
	border-radius:5px;
}

QRadioButton:hover {
	background-color: rgba(230,230,230, 1.0);
	border: 0px; 
	border-radius:5px;
}

QRadioButton:checked:hover {
	background-color: rgba(245,245,245, 1.0);
	border: 0px; 
	border-radius:5px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
}
QRadioButton::indicator::unchecked {
    image: url(:/main/img/radio_off1.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/main/img/radio_off1.png);
}

QRadioButton::indicator::checked {
    image: url(:/main/img/radio_on2.png);
}</string>
    </property>
    <property name="text">
     <string>Fast</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_precise">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>135</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadioButton:checked {
	background-color: rgba(245,245,245, 1.0);
	border: 0px; 
	border-radius:5px;
}

QRadioButton:hover {
	background-color: rgba(230,230,230, 1.0);
	border: 0px; 
	border-radius:5px;
}

QRadioButton:checked:hover {
	background-color: rgba(245,245,245, 1.0);
	border: 0px; 
	border-radius:5px;
}

QRadioButton::indicator {
    width: 20px;
    height: 20px;
}
QRadioButton::indicator::unchecked {
    image: url(:/main/img/radio_off1.png);
}

QRadioButton::indicator:unchecked:hover {
    image: url(:/main/img/radio_off1.png);
}

QRadioButton::indicator::checked {
    image: url(:/main/img/radio_on2.png);
}</string>
    </property>
    <property name="text">
     <string>Slow</string>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>135</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Roboto Light</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Mode:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <family>Roboto</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionEMBO_Help"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/main/img/info.svg</normaloff>:/main/img/info.svg</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
  </action>
  <action name="actionEMBO_Help">
   <property name="icon">
    <iconset resource="../../resources/resources.qrc">
     <normaloff>:/main/img/doc.png</normaloff>:/main/img/doc.png</iconset>
   </property>
   <property name="text">
    <string>EMBO Help</string>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
